pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -578,21 +574,34 @@ public CompletableFuture<Void> closeAsync() {
         }
     }
 
+    private void close(AutoCloseable closeable, ScheduledExecutorService shutdownExecutor) {
```

Comment:
Add a param to represent which resource to close. If failed, print it in log info.

Reply:
private void close(AutoCloseable closeable, String resourceName, ScheduledExecutorService shutdownExecutor)

Reply:
updated.